server/code/setup/cpdb

```diff
@@ -33,7 +33,7 @@ JBOSS_CLASSPATH = "/var/lib/jbossas/server/production/deploy/candlepin.war/WEB-I
 TOMCAT_CLASSPATH = "/var/lib/" + TOMCAT + "/webapps/candlepin/WEB-INF/classes/"
 
 def run_command(command):
-    (status, output) = commands.getstatusoutput(command)
+    (status, output) = subprocess.getstatusoutput(command)
```

Comment:
This is tricky, as the return values of `commands.getstatusoutput` and `subprocess.getstatusoutput` actually differ:

# `commands.getstatusoutput(cmd)`

Execute the string `cmd` in a shell with `os.popen()` and return a 2-tuple `(status, output)`. `cmd` is actually run as `{ cmd ; } 2>&1`, so that the returned output will contain output or error messages. A trailing newline is stripped from the output. The exit status for the command can be interpreted according to the rules for the C function `wait()`.

# `subprocess.getstatusoutput(cmd)`

Return `(exitcode, output)` of executing `cmd` in a shell.

Execute the string `cmd` in a shell with `Popen.check_output()` and return a 2-tuple `(exitcode, output)`. The locale encoding is used; see the notes on Frequently Used Arguments for more details.

A trailing newline is stripped from the output. The exit code for the command can be interpreted as the return code of subprocess.

The function now returns `(exitcode, output)` instead of `(status, output)` as it did in Python 3.3.3 and earlier. exitcode has the same value as `returncode`.

# difference

The difference is that in Python2 we had to pass `status` through [`os.WEXITSTATUS`](https://docs.python.org/2/library/os.html#os.WEXITSTATUS) to get the real exit code, whereas we don't have to do this since Python 3.4. That said, the old code is *technically* wrong (as it won't catch all possible error conditions), but the new one is correct.

See https://github.com/Katello/katello-client-bootstrap/commit/5e7e0130d4581a07beb5c0ecc58c2f3653781300 and https://github.com/Katello/katello-client-bootstrap/commit/d1c308a3b8648cc4c2b12043fdcd7f22371bcd56 for an example how we used that in `katello-client-bootstrap`.